### PR TITLE
DTGB-426: Did some refactoring.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/00-mixins/general/_mixins.scss [hunk 1]
```diff
@@ -276,18 +214,50 @@
...
+/// @group tools
+/// @access private
+/// @author Gert-Jan Meire
+///
 @mixin small-text {
   font-size: .6rem;
 }
 
+///
+/// Defines medium text.
```
Comment (delrueba) [inline styleguide/components/00-mixins/general/_mixins.scss:230]: Should be obsolete.
If medium text is not small or large: it's default.

File: styleguide/components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss [hunk 2]
```diff
@@ -74,13 +74,6 @@
...
 
-    &::before {
-      @include triangle(top, 10px, 10px, color('white'));
-      position: absolute;
-      top: -10px;
-      right: 10px;
-    }
-
     h3 {
       margin-bottom: 5px;
```
Comment (delrueba) [inline styleguide/components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss]: rem

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/00-mixins/general/_mixins.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/00-mixins/general/_mixins.scss:230]: As discussed to be changed later.

File: styleguide/components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss]: Fixed.

**Review by delrueba — APPROVED** [review]

**Pull request merged**